notebooks/exploratory_data_analysis.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    for query_file_path in glob.glob(f'{QUERIES_FOLDER_PATH}/*{QUERY_FILE_EXTENSION}'):
</pre><p>I recomand to use pathlib instead of glob.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ted-data-eu/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Tanks for recommendation, change is done.

```python
notebook_path = Path.cwd()
for query_file_path in notebook_path.glob(f'{QUERIES_FOLDER_PATH}/*{QUERY_FILE_EXTENSION}'):
```